plugins/agent_types/AcquiaLiftLearn.inc

```diff
@@ -80,22 +80,25 @@ class AcquiaLiftLearn extends PersonalizeAgentBase implements AcquiaLiftLearning
   public function getAgentSyncOperations($option_sets, $new_goals, $old_goals = array()) {
     // Currently only a single option set is supported.
     $option_set = reset($option_sets);
+    // If the test agent has a site name, use this as a prefix for goals and
+    // decision sets.
+    $prefix = isset($this->data['site_name']) ? $this->data['site_name'] . '-' : '';
```

Comment:
This is the second place I've seen the prefix defined (https://github.com/acquia/acquia_lift/blob/unique-names/acquia_lift.module#L940) - could it be encapsulated somewhere for those of us who forget if it's a hyphen or an underscore, etc.?